src/snyk/common/constants/views.ts

```diff
@@ -13,12 +13,11 @@ export const SNYK_VIEW_ANALYSIS_IAC = 'snyk.views.analysis.configuration';
 // Having multiple boolean contexts instead of a single context
 // with multiple values helps us to avoid flickering UI.
 export const SNYK_CONTEXT = {
+  NOT_LOADING: 'notLoading', // default to loading state (notLoading = false when boolean is initialized)
```

Comment:
I'm new to this project so please dismiss this comment if not relevant 🙏 

I usually find negatives in names for standalone variables a bit confusing to follow. For context/justification about my comment, I'm basing my thinking on articles like [this](https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables).

I can follow that we use it to display a view or not, right?

```
    // initialize context correctly
    // see package.json when each view is shown, based on context value
    await this.contextService.setContext(SNYK_CONTEXT.NOT_LOADING, false);
```

Does it make sense to call it `showSomething`/`showView`?

Reply:
We need a state where initialization with false is fine :). Else I'm open for any name.

Reply:
I'm not saying to not use a variable to hold the state. I was talking about naming conventions but this is really minor so don't worry about my comment 🙏